app/qml/main.qml

```diff
@@ -898,7 +898,6 @@ ApplicationWindow {
     {
       if ( projectFullName === __activeProject.projectFullName() )
       {
-        syncInProgressAnimation.stop()
```

Comment:
Is this not needed anymore? `syncButton.iconRotateAnimationRunning = false`

Reply:
Apparently not. `syncButton.iconRotateAnimationRunning` is only set to true by the sync manager when `__syncManager::onSyncStarted` and when `__syncManager::hasPendingSync`, which can't happen without authing first.